install/host_preparation.adoc

```diff
@@ -774,15 +774,16 @@ of log files.
 |Sets the maximum number of log files to be kept per host.
 |===
 
-For example, to set the maximum file size to 1MB and always keep the last three
-log files, edit the *_/etc/sysconfig/docker_* file to configure `max-size=1M`
-and `max-file=3`:
-
+. To configure the log file, edit the *_/etc/sysconfig/docker_* file.
+For example, to set the maximum file size to 1MB and always retain the last three
+log files, append `max-size=1M` and `max-file=3` to the `OPTIONS=` line,
+being sure to keep them within the single quotes:
```

Comment:
s/being sure to keep them within the single quotes/ensuring that the values maintain the single quotation mark formatting